scripts/foosbot.coffee

```diff
@@ -694,6 +695,58 @@ returnFromCleanseRespond = (res) ->
     res.reply "Welcome back! Now go kick some ass"
 
 
+getTeamStats = (playerOneName, playerTwoName) ->
+  stats = {
+    "wins": 0,
+    "losses": 0,
+    "goalsFor": 0,
+    "goalsAgainst": 0
+  }
+
+  for finishedGame in finishedGames
+    teamOne = [fgame["team1"]["player1"], fgame["team1"]["player2"]]
+    teamTwo = [fgame["team2"]["player1"], fgame["team2"]["player2"]]
+
+    if playerOneName in teamOne && playerTwoName in teamOne
+      myTeam = finishedGame["team1"]
+      otherTeam = finishedGame["team2"]
+    else if playerOneName in teamTwo && playerTwoName in teamTwo
+      myTeam = finishedGame["team2"]
+      otherTeam = finishedGame["team1"]
+    else
+      continue
+
+    if myTeam["score"] > otherTeam["score"]
+      stats["wins"] += 1
+    else
+      stats["losses"] += 1
+    stats["goalsFor"] += myTeam["score"]
+    stats["goalsAgainst"] += otherTeam["score"]
+
+  return stats
+
+
+teamStatsRespond = (res) ->
+  playerOneName = if res.match[1] == 'me' then res.message.user.name else res.match[1].trim()
+  playerTwoName = if res.match[2] == 'me' then res.message.user.name else res.match[2].trim()
+
+  if playerOneName == playerTwoName
+    return
```

Comment:
I'd suggest `res.send`ing a message here before returning, so the user knows this isn't allowed.